app/routes/experiments/info/preview.js

```diff
@@ -1,31 +1,27 @@
 import Ember from 'ember';
 import AuthenticatedRouteMixin from 'ember-simple-auth/mixins/authenticated-route-mixin';
 
+import ExpPlayerRouteMixin from 'exp-player/mixins/exp-player-route';
 
-export default Ember.Route.extend(AuthenticatedRouteMixin, {
-    model(params) {
-        let experiment = this.modelFor('experiments.info');
-        let session = this.store.createRecord(experiment.get('sessionCollectionId'), {
+export default Ember.Route.extend(AuthenticatedRouteMixin, ExpPlayerRouteMixin, {
+    _getExperiment() {
+        return new Ember.RSVP.Promise((resolve) => {
+            resolve(this.modelFor('experiments.info'));
+        });
+    },
+    _getSession(params, experiment) {
+        var session = this.store.createRecord(experiment.get('sessionCollectionId'), {
             experimentId: experiment.id,
             profileId: 'tester0.prof1', // TODO fetch from service
```

Comment:
This was always a TODO- but since we're actually writing the service anyway, care to do the honors and wire it up? 

Aside from that: as far as I can tell this PR would be largely internal. Let me know if there is a specific new/changed behavior that I should look for in functional review, when confirming that it works as expected.